setup.py

```diff
@@ -0,0 +1,20 @@
+# beprof setup.py
+
+from distutils.core import setup
+
+setup(
```

Comment:
Check what happens if user doesn't have numpy installed.
Its bad idea to instally numpy/scipy using pip, especially on Windows.
Investigate how to put dependency on numpy in setup.py.
It seems there is some discussion about it on stackoverflow: http://stackoverflow.com/questions/25753276/install-numpy-pandas-as-dependency-in-setup-py